src/command/QueueCommands.cxx

```diff
@@ -125,6 +125,14 @@ CommandResult
 handle_addid(Client &client, Request args, Response &r)
 {
 	const char *uri = args.front();
+	if (StringIsEqual(uri, "/"))
```

Comment:
This is an illegal value. This workaround is bad here, and should only be in `add` because only `add` had this historically. This was made illegal before `addid` was even implemented.

Reply:
Yes, you are right. Bad changes.